controllers/provider-aws/pkg/controller/controlplane/valuesprovider.go

```diff
@@ -188,6 +188,28 @@ func (vp *valuesProvider) GetStorageClassesChartValues(
 	return nil, nil
 }
 
+// GetControlPlaneExposureChartValues updates the AWS ELB health check to SSL and deploys the aws-lb-readvertiser.
+// https://github.com/gardener/aws-lb-readvertiser
+func (vp *valuesProvider) GetControlPlaneExposureChartValues(
+	ctx context.Context,
+	cp *extensionsv1alpha1.ControlPlane,
+	cluster *extensionscontroller.Cluster,
+) (map[string]interface{}, error) {
+
+	replicas := 0
+	if extensionscontroller.IsHibernated(cluster.Shoot) {
+		replicas = 1
+	}
+
+	return map[string]interface{}{
+		"domain":   "", // TODO loadBalancerIngress is computed after kube-apiserver service is ready and it is assigned to botanist.operation.APIServerAddress
```

Comment:
You could use [this function](https://github.com/gardener/gardener-extensions/blob/ba20cf1c9d2aeb21c38bdc8606f0cbdfc4298120/controllers/provider-alicloud/pkg/webhook/controlplaneexposure/ensurer.go#L60-L64).